packages/playground/src/weblets/tf_algorand.vue

```diff
@@ -194,6 +196,7 @@
 import { computed, type Ref, ref, watch } from "vue";
 
 import Network from "../components/networks.vue";
+import NodeSelector from "../components/node_selection.vue";
```

Comment:
i think if we renamed the file to -> `manual_node_selection` will be better since we have another node selector component